monitoring/monitoring_on.go

```diff
@@ -29,7 +29,12 @@ func GetPromInterceptors(cfg *PrometheusConfig) ([]grpc.UnaryServerInterceptor,
 	}
 
 	if cfg.PerfHistograms {
-		opt := grpc_prometheus.WithServerHandlingTimeHistogram()
+		// Set the histogram buckets in seconds.
+		histogramBuckets := []float64{1, 5, 10, 60, 120, 240, 600, 1200}
```

Comment:
Should we keep some of the smaller (sub-second) values? Like `0.01, 0.1, 0.5`? Because one second is quite long for any gRPC call, so we wouldn't notice faster calls to get slower until they actually hit a full second (I hope you know what I mean).

Reply:
Sure why not, I was thinking of the data cost, but we can scale back later if it seems too much.